app/src/main/java/com/example/hossam/parashotApp/dataLayer/apiData/ApiClient.java

```diff
@@ -37,4 +38,15 @@ public static Retrofit getClient() {
         }
         return retrofit;
     }
+
+    public static Retrofit getClientForGoogle() {
```

Comment:
This new Retrofit object is not needed as retrofit ignore base URL if you use full URL in `@Post` or `@Get`